projects/ontimize-web-ngx-report/src/lib/components/report/o-report-detail/o-report-detail.component.scss

```diff
@@ -0,0 +1,24 @@
+.report-detail {
+  .vertical-margin-10 {
+    margin: 10px 0;
+  }
+
+  .form-field {
+    padding-bottom: 6px;
+  }
+}
+
+.o-column.vertical-margin-10.o-column {
+  margin: 10px 0;
+}
+
+.o-button {
```

Comment:
This button customization should has a more specific selector that affects only the button of this form because of when using report-detail in a final app it can modify other buttons.